### ISSUE TITLE
GPU not displayed when using Nvidia proprietary driver

### ISSUE THREAD

**Issue by stuartsoft** [KittyKatt/screenFetch#177]
Hi there KittyKatt! I'm running Mint 17.1 with an Nvidia gtx 970 under the proprietary driver since there seems to be no opensource drivers available for it yet. I ran screenFetch and it could not properly detect the gpu using lspci. I've been looking at the script to see if I can see exactly what's wrong, but just wanted to give you a heads up for the time being!
![screenshot from 2015-02-03 13 53 10](https://cloud.githubusercontent.com/assets/6208902/6026871/1e429ae4-abac-11e4-933f-2e80fbcf1222.png)
![screenshot from 2015-02-03 13 53 33](https://cloud.githubusercontent.com/assets/6208902/6026870/1e404b5e-abac-11e4-8085-baa32f0a05d9.png)

**Comment by KittyKatt** [comment]
That's interesting. I wonder what's causing that.

**Comment by cgoldberg** [comment]
fwiw, I'm running Ubuntu 14.04, with nvidia proprietary driver (version 331), and screenfetch correctly identifies my GeForce GTX 650 Ti Boost.

**Comment by stuartsoft** [comment]
Perhaps it's an issue with the newer proprietary drivers then? I'm on 343.36. LM 17.1 is based on Ubuntu 14.04LTS  so it definitely sounds like a driver specific issue.

**Comment by JammySTB** [comment]
Getting the same issue with a 970 on the 346.37 driver.

**Comment by KittyKatt** [comment]
I'll have to devise a few tests for you guys. Could someone that is currently experiencing this issue give me a full output of `lspci`, `glxinfo`, and `nvidia-smi -q` that'd be great.

**Comment by stuartsoft** [comment]
So I just went and recloned the repo and the current commit (e0d9302) works fine and detects my Asus GTX 970 as "GeForce GTX 970/PCIe/SSE2". It should be noted that I'm using the exact same driver (343.36 proprietary) on the same machine as the original post, so whatever you've done in the past 2 months seems to have fixed it :)

I then checked out the commit I was probably using at the original time of this thread, (1f7bd01) from Jan 24 and sure enough the gpu didn't show up. Below are my  results for lspci, glxinfo and nvidia-smi -q

[lspci pastebin](http://pastebin.com/9JKZAENv)
[glxinfo pastebin](http://pastebin.com/N0BnAXgm)
[nvidia-smi -q pastebin](http://pastebin.com/SF2hFykq)

Thanks so much KittyKatt, keep doin what you do!
